### PR TITLE
issue/326 | update: the homepage to explain how builds work along wit…

### PR DESCRIPTION
**Pull request by staghouse**
…h versioning. update: routing to be more descriptive. fix: anchoring on the homepage hiding headers.

Close #336

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -17,13 +17,21 @@
...
 ---
 
-# MTGJSON
+# What Is MTGJSON?
 
-## Our Mission
+## How It Works
 
-MTGJSON is an open-source project that catalogs all Magic: The Gathering cards in a portable format. A dedicated group of fans maintains and supplies data for a variety of projects and sites in the community.
+MTGJSON is an open-source project that catalogs all Magic: The Gathering cards in a portable format. A dedicated group of fans maintains and supplies data for a variety of projects and sites in the community. Using an aggregation process we fetch data between multiple resources and approved partners, and combine all this data in to various JSON files that you can learn about and download from this website.
```
Comment (tooomm) [inline docs/README.md]: "into"?

File: docs/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 {
-  "title": "MTGJSON",
+  "title": "Portable formats for Magic: The Gathering card data",
   "meta": [
     {
       "name": "description",
```
Comment (tooomm) [inline docs/README.md]: Isn't that more of a description and should replace that, while the title stays MTGJSON?

Comment (staghouse) [reply docs/README.md]: Yes and No. Yes it is, but no its not when thats the first thing people see as we are the top result for google.com

Comment (staghouse) [reply docs/README.md]: You do bring up a good point about refining our SEO which I will open a ticket for.

**Pull request merged**